### ISSUE TITLE
[Suggestion] format error snippet

### ISSUE THREAD

**Issue by ldelossa** [fatih/vim-go#1760]
Looking at the provided snippets, I would find this to be useful instead of the current `errn,` 

```
❯ git diff origin/master
diff --git a/gosnippets/UltiSnips/go.snippets b/gosnippets/UltiSnips/go.snippets
index 9a9f546..87a7b37 100644
--- a/gosnippets/UltiSnips/go.snippets
+++ b/gosnippets/UltiSnips/go.snippets
@@ -169,7 +169,7 @@ endsnippet
 # error multiple return
 snippet errn, "Error return with two return values" !b
 if err != nil {
-       return ${1:nil}, err
+       return ${1:nil}, ${2:err}
 }
 ${0}
 endsnippet
```

Pushing branches are not allowed so could not open a PR.

**Comment by Carpetsmoker** [comment]
> Pushing branches are not allowed so could not open a PR.

You'll have to fork the repo: https://help.github.com/articles/fork-a-repo/

**Comment by ldelossa** [comment]
Not a problem

**Comment by Carpetsmoker** [comment]
Fixed in  #1761
